README.md

```diff
@@ -1 +1,164 @@
-# leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="/images/eldonazul1.jpeg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>ELDON RATZLAFF LEADERBOARD PROYECT</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 ELDON RATZLAFF LEADERBOARD <a name="about-project"></a>
+
+> PROYECTS AND SKILS FROM ELDON RATZLAFF
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> LEADERBOARD PROYECT. Uses GitHub, HTML, CSS, js, API, Linters
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JS</a></li>
+    <li><a href="#">API</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **It shows you my projects**
+- **It shows you my skills**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo of my page"></a>
+
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://eldonratzlaff.github.io/leaderboard/)
```

Comment:
- [OPTIONAL] This link is broken so I will highly recommend you to consider fixing it.